libpod/container_internal.go

```diff
@@ -1290,3 +1293,23 @@ func getExcludedCGroups() (excludes []string) {
 	excludes = []string{"rdma"}
 	return
 }
+
+// namedVolumes returns named volumes for the container
+func (c *Container) namedVolumes() ([]string, error) {
+	var volumes []string
+	spec, err := c.specFromState()
```

Comment:
Use c.config.spec instead, specFromState is expensive and pointless, we don't add any named volumes to the generated spec on disk

Reply:
That is what I had originally but the volume is not listed in the spec, it is only listed when I load from state.

Reply:
Wait, what? We're not adding these when we generate the spec... At least, we shouldn't be... This is in a really bad spot if we can't find volumes without the running spec